Dockerfile

```diff
@@ -5,7 +5,7 @@
 # Purpose: Compiles the frontend
 # Notes:
 #  - Does NPM stuff with Typescript and such
-FROM --platform=$BUILDPLATFORM node:16-bullseye-slim AS compile-frontend
+FROM --platform=$BUILDPLATFORM docker.io/node:16-bullseye-slim AS compile-frontend
```

Comment:
Use [`node:16-bookworm-slim`](https://hub.docker.com/layers/library/node/16-bookworm-slim/images/sha256-1362db52e34df6d0fd67619257c4169c0b39c6c0e8d3ec3c4b3c843bf7e3e2f0?context=explore) instead to keep it consistent to `python:3.9-slim-bookworm`?
But in fact that should not make much difference...

Reply:
Good point.  I wouldn't expect any difference, but consistent is best.